src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -47,7 +48,27 @@
 }
 
 
-def histogram(polygons):
+def histogramPoll(jobId):
+    host = settings.GEOP['host']
+    port = settings.GEOP['port']
+    path = '/jobs/' + jobId
+    conn = httplib.HTTPConnection("%s:%s" % (host, port))
+
+    # Try for about two minutes ...
+    seconds = 1
+    for i in range(9):
+        time.sleep(seconds)
+        try:
+            conn.request('GET', path)
+            data = json.loads(conn.getresponse().read())
+            if data['status'] == 'OK':
+                return data['result']
+        finally:
+            seconds *= 1.618
```

Comment:
Ah. SciPy has this setup as a constant. I think it may be a good idea to do something similar here.